JSDemos/Demos/PivotGrid/RunningTotals/description.md

```diff
@@ -1 +1,3 @@
-The PivotGrid component is&nbsp;able to&nbsp;display running total values next to&nbsp;summary cells. The component can perform cross-group accumulation of&nbsp;running total values or&nbsp;accumulate them within each group separately.
\ No newline at end of file
+The PivotGrid can calculate running totals by rows or columns. To specify the direction, set a field's [runningTotal](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#runningTotal) property to *"row"* or *"column"*. This property is available only for data fields.
```

Comment:
> To specify the direction

This phrase indirectly implies a choice between rows and columns as possible sources for running total calculations. I would have rephrased this in a more direct manner.